test/unit/rules/no-element-event-actions-test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const generateRuleTests = require('../../helpers/rule-test-harness');
-const ERROR_MESSAGE = require('../../../lib/rules/lint-no-element-event-actions').ERROR_MESSAGE;
+const ERROR_MESSAGE = require('../../../lib/rules/no-element-event-actions').ERROR_MESSAGE;
```

Comment:
this seems wrong 🤔

Reply:
how so?

Reply:
because `../../../lib/rules/no-element-event-actions../../../lib/rules/no-element-event-actions` looks like something in your find-and-replace went wrong 😉

Reply:
but now I'm confused because the notification email showed:

```
-const ERROR_MESSAGE = require('../../../lib/rules/lint-no-element-event-actions').ERROR_MESSAGE;
+const ERROR_MESSAGE = require('../../../lib/rules/no-element-event-actions').ERROR_MESSAGE;
```

is GitHub displaying broken diffs to me right now?! 🤔

Reply:
I didn't use f&r because I was trying to avoid screwing up- but anyway here's what I see, maybe we can figure it out this way: 
![image](https://user-images.githubusercontent.com/4587451/69549104-3fcd8f00-0f5e-11ea-9637-a5386a165cce.png)

Which seems accurate (and the tests suggest the same, since they are passing). 

Is there something else I am missing?

Reply:
I think I figured it out. I'm using https://addons.mozilla.org/de/firefox/addon/octolinker/ and it seems that their latest version has a bug which sometimes duplicates text... 😱